app/src/main/java/org/coepi/android/cen/RealmCenLastKeysCheckDao.kt

```diff
@@ -0,0 +1,29 @@
+package org.coepi.android.cen
+
+import io.realm.Sort.DESCENDING
+import io.realm.kotlin.createObject
+import io.realm.kotlin.where
+import org.coepi.android.repo.RealmProvider
+
+class RealmCenLastKeysCheckDao(private val realmProvider: RealmProvider) {
```

Comment:
Why use the database? Seems like the preferences would be enough? (see `Preferences` component)